darwin/exporter/formats/darwin_1_0.py

```diff
@@ -0,0 +1,134 @@
+import json
+from pathlib import Path
+from typing import Any, Dict, Iterable, List, Union
+
+import darwin.datatypes as dt
+from darwin.exporter.formats.numpy_encoder import NumpyEncoder
+from darwin.version import __version__
```

Comment:
why are we importing `__version__` here?

Reply:
I have no clue, probably IDE acting out. Removed.